library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
Oh, and I think it is called "bootstrap", not booze-trap (function include_boostrap) 😜 

I think I like the "call_any_libraries you want(1,0,0,0,2,1,1,1,0); function idea myself.